backlogger.py

```diff
@@ -275,7 +275,7 @@ def trigger_webhook(state, bad_queries):
     parser.add_argument(
         "--output", choices=["markdown", "influxdb"], default="markdown"
     )
-    parser.add_argument("--reminder-comment-on-issues", action="store_false")
+    parser.add_argument("--reminder-comment-on-issues", action="store_true")
```

Comment:
The default is supposed to be false, though.

Reply:
Hrm. [According to the docs](https://docs.python.org/3/library/argparse.html#action) the defaults are flipped? I guess it means reminder comments would only have been "sent" when they were not supposed to.

Reply:
So the alternative is probably *const*? Not sure either is more readable to be honest, so I guess either way is fine by me.

```suggestion
    parser.add_argument("--reminder-comment-on-issues", action="store_const", const=True)
```